packages/shared/src/nomination-pools.ts

```diff
@@ -321,6 +368,30 @@ async function nominationPoolLifecycleTest(relayChain, addressEncoding: number)
 
   await check(nominationPoolWithCommission.commission).toMatchObject(newCommissionData)
 
+  /**
+   * Nominate a validator set
+   */
+
+  const validators = await getRandomValidators(relayClient.api, 100)
+
+  const nominateTx = relayClient.api.tx.nominationPools.nominate(nomPoolId, validators)
+  const nominateEvents = await sendTransaction(nominateTx.signAsync(defaultAccounts.charlie))
+
+  await relayClient.dev.newBlock()
+
+  // `nominate` does not emit any events from `staking` or `nominationPools` as of
+  // Jan. 2025. #7377 will fix this.
```

Comment:
full github link is better than issue number because it is hard to tell if this is issue in this repo, or polkadot-sdk, or runtime